core/trino-main/src/main/java/io/trino/execution/DropMaterializedViewTask.java

```diff
@@ -57,7 +57,7 @@ public String getName()
         Optional<ConnectorMaterializedViewDefinition> view = metadata.getMaterializedView(session, name);
         if (view.isEmpty()) {
             if (!statement.isExists()) {
-                throw semanticException(TABLE_NOT_FOUND, statement, "View '%s' does not exist", name);
+                throw semanticException(TABLE_NOT_FOUND, statement, "Materialized view '%s' does not exist", name);
```

Comment:
nit: we could also have a engine only test (like we do for selects, see: `TestAnalyzer`)